app/src/main/AndroidManifest.xml

```diff
@@ -217,7 +217,8 @@
 
         <activity
             android:name=".ui.publicShare.PublicShareActivity"
-            android:configChanges="orientation|screenSize|layoutDirection" />
+            android:configChanges="orientation|screenSize|layoutDirection"
+            android:excludeFromRecents="true" />
```

Comment:
Why?

Reply:
Because I don't want this activity to make the app appearing in the "recent apps" on the user's phone
Just opening a link shouldn't count as a real use of the app